Dockerfile

```diff
@@ -1 +1,41 @@
+ARG ARCH="amd64"
+ARG OS="linux"
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
+LABEL maintainer="Armedia, LLC"
 
+ARG ARCH="amd64"
+ARG OS="linux"
+ARG VER="2.28.1"
+ARG PKG="prometheus"
+ARG SRC="${PKG}-${VER}.${OS}-${ARCH}"
+
+RUN curl \
+		-L "https://github.com/prometheus/${PKG}/releases/download/v${VER}/${SRC}.tar.gz" \
+		-o package.tar.gz && \
```

Comment:
We decided to store the artifacts in our own S3 bucket: `arkcase-container-artifacts`
We also have a `get-artifacts.sh` script in each repo to download the artifacts from the S3 bucket prior to building the Docker image (this is because otherwise we would need to have the awscli installed inside the container during the Docker build, along with some credentials and that was seen as too complicated).

Reply:
I see the value in this, but the intent here is to get it from the source - i.e. "unadulterated", if you will...

Reply:
What Jon suggested is that we should be able to build our images even if the original sources are not available (server down or other things). Also, if we pull a lot of images from Docker Hub (esp. from the CI/CD pipelines), we might hit Docker Hub rate limits.

So what we do is download the "unadulterated" package and upload it to the S3 bucket. Then the `get-artifacts.sh` script downloads the artifacts from the S3 bucket.